build/dockerfiles/apk.Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN apk upgrade --update-cache \
     && adduser -S kong \
     && addgroup -S kong \
     && mkdir -p "${KONG_PREFIX}" \
+    && mkdir /var/run/kong \
+    && chown kong:0 /var/run/kong \
```

Comment:
Do we need to `chmod` here for full parity with the systemd-tmpfiles setup?

Reply:
Added to mkdir command